dapp/src/components/sidePanel/SidePanelInsuranceMessage.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from 'react'
+import { fbt } from 'fbt-runtime'
+import Link from 'next/link'
+
+const SidePanelInsuranceMessage = () => {
+  const [show, setShow] = useState(true)
+  const localStorageKey = 'HideSidePanelInsuranceMessage'
+
+  useEffect(() => {
+    setShow(!(localStorage.getItem(localStorageKey) === 'true'))
+  }, [])
+
+  return (
+    <>
+      {show && (
+        <div className="side-panel-message d-flex flex-column align-items-center justify-content-center">
+          <a
+            className={`dismiss-link`}
+            onClick={(e) => {
+              e.preventDefault()
+              e.stopPropagation()
+              localStorage.setItem(localStorageKey, 'true')
+              setShow(false)
+            }}
+          >
+            ×
+          </a>
+          <img className="insurance-icon" src="/images/ousd-shield-icon.svg" />
+          <div>
+            {fbt(
+              'Get optional smart contract insurance for your OUSD',
+              'Get optional smart contract insurance for your OUSD'
+            )}
+          </div>
+          <Link href="#">
```

Comment:
In case Learn more is a link to another site would be cool to open in another window by adding `target="_blank"`

Reply:
Updated